ReactWindows/ReactNative/Views/Text/ReactTextViewManager.cs

```diff
@@ -1,16 +1,19 @@
 ﻿using ReactNative.UIManager;
+using System;
+using Windows.UI;
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Controls;
-using Windows.UI.Xaml.Documents;
+using Windows.UI.Xaml.Media;
 
 namespace ReactNative.Views.Text
 {
     /// <summary>
     /// The view manager for text views.
     /// </summary>
-    public class ReactTextViewManager : BaseViewManager<TextBlock, ReactTextShadowNode>
+    public class ReactTextViewManager : BaseViewManager<Border, ReactTextShadowNode>
```

Comment:
Can we keep this as text block? I get that this would allow background color and border, but if it's not natively supported on TextBlocks in UWP, we shouldn't pretend it is by wrapping this in a border (we could just as easily back the text with a View in JavaScript).